committeeoversightapp/views.py

```diff
@@ -91,8 +107,8 @@ def filter_queryset(self, qs):
 
     def prepare_results(self, qs):
         json_data = []
-        edit_string = "<a href=\"/edit/{}\"><i class=\"fas fa fa-pencil-alt\" id=\"edit-icon\"></i></a>"
-        delete_string = "<a href=\"/delete/{}\"><i class=\"fas fa fa-times-circle\" id=\"delete-icon\"></i></a>"
+        edit_string = "<a href=\"edit/{}\"><i class=\"fas fa fa-pencil-alt\" id=\"edit-icon\"></i></a>"
```

Comment:
the edit and delete buttons are working in the hearing listing, but not from the category detail view. if they should be visible in that view, you'll want to start your url with a slash (`/hearings/edit/`, iirc?) so the link urls aren't relative to the current request location. https://webmasters.stackexchange.com/questions/56840/what-is-the-purpose-of-leading-slash-in-html-urls

Reply:
Ah true thank you!